packages/cozy-procedures/src/components/overview/helpers.js

```diff
@@ -0,0 +1,86 @@
+import formatDate from 'date-fns/format'
+import subMonths from 'date-fns/sub_months'
+import startOfMonth from 'date-fns/start_of_month'
+import pick from 'lodash/pick'
+
+/**
+ * createJsonFile - upload a json file containing the given data
+ *
+ * @param {string} filename - The json file name
+ * @param {string} destinationId - The id of the destination directory
+ * @param {string} content - The file content
+ * @return {object} Information about the jsonFile (id and name)
+ */
+export const createJsonFile = async (
+  client,
+  filename,
+  destinationId,
+  content
+) => {
+  const file = new File([content], filename)
+  const resp = await client
+    .collection('io.cozy.files')
+    .createFile(file, { dirId: destinationId })
+
+  return resp.data
+}
+
+export const fetchCheckingsAccounts = async client => {
```

Comment:
`fetchCheckingsLikeAccounts` ? It does not sound good but IMHO it conveys better the intention that there are more than checkings accounts that are considered.